wagtail/admin/utils.py

```diff
@@ -20,3 +23,36 @@ def get_admin_base_url():
         admin_base_url = settings.BASE_URL
 
     return admin_base_url
+
+
+def get_block_usage():
+    """
+    Scans all StreamFields in the database and counts the number of times each block type has been used.
+
+    Returns:
+        A list of 2-tuples (block, usage_count) where 'block' is an instance of a Block and the usage_count
+        is the number of times that block has been used.
+    """
+    blocks = {}
+    counter = Counter()
+
+    for model in apps.get_models():
+        stream_fields = [
+            field
+            for field in model._meta.get_fields()
+            if isinstance(field, StreamField)
+        ]
```

Comment:
There's an existing method to get these
```suggestion
        stream_field_names = model.get_streamfield_names()
        stream_fields = [
            model._meta.get_field(field)
            for field in stream_field_names
        ]
```